pkgs/top-level/ocaml-packages.nix

```diff
@@ -607,6 +607,8 @@ let
 
     lambdaTerm = callPackage ../development/ocaml-modules/lambda-term { };
 
+    ledit = callPackage ../development/ocaml-modules/ledit { };
```

Comment:
The `ledit` package only provides a binary program whose functionality does not depend on the version of OCaml used for compiling it. Therefore, it does not belong to the `ocamlPackages` package set. Please move it to `all-packages.nix`.

Reply:
Okay, I wasn't sure how packages were split between ocamlPackages, all-packages, etc...
Thanks for the clarifications
I moved ledit to all-packages.nix